execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -345,25 +345,26 @@ impl TransactionGenerator {
     }
 
     /// Generates transactions for random pairs of accounts.
-    pub fn gen_transfer_transactions(&mut self, block_size: usize, num_blocks: usize) {
+    pub fn gen_transfer_transactions(&mut self, block_size: usize, num_blocks: usize, transactions_per_sender: usize) {
         for _ in 0..num_blocks {
-            let transactions: Vec<_> = (0..block_size)
+            let transactions: Vec<_> = (0..(block_size / transactions_per_sender))
```

Comment:
Hmm this will cause block_size to be inaccurate when it is not divisible by the `transactions_per_sender`.

Reply:
As discussed, let's put a TODO here and we can land it for now and fix it later.